charts/alfresco-repository/values.yaml

```diff
@@ -0,0 +1,397 @@
+repository:
```

Comment:
That level of values should be removed and its content moved to the root context.